scss/_settings_colors.scss

```diff
@@ -108,7 +124,7 @@ $states: (
 // --border-low-contrast  - low contrast version of border color, to be used when border needs more visibility (separators)
 
 // Light theme
-$colors--light-theme--text-default: #000 !default;
+$colors--light-theme--text-default: $vf-color-font-default !default;
```

Comment:
Why "font" and not "text"?

Reply:
Looking at the sheet from @dgtlntv, they also used `text` for text colour variables: https://docs.google.com/spreadsheets/d/1AY6BLFU-Cnj48ggHUnNVlWRxwWZ6Z-66s7jxJC1HLSU/edit?gid=1738212977#gid=1738212977